syncthing/Caddyfile

```diff
@@ -0,0 +1,16 @@
+example.com {
+        gzip
+
+	# redirect all requests (include trailing '/')
```

Comment:
It might be more accurate for this comment to state that it "ensures all requests include trailing slashes".